api/client.ts

```diff
@@ -180,6 +180,7 @@ function connectAndLogin(url: string, credentials, options) {
         }
       }
       reject("cannot connect to model after redirection");
+      return;
```

Comment:
This needs to go up after 159. 

Because if the connection to Juju fails then after rejecting we don't want it to continue to try to connect to servers that don't exist.

Reply:
👍🏻